syntax = "proto3";
option optimize_for = LITE_RUNTIME;

message EffectiveMsg {
    int32 level = 1;
    int64 timestamp = 2;
    int32 pid = 3;
    int32 tid = 4;
    int32 line = 5;
    int32 column = 6;
    string file_name = 7;
    string func_name = 8;
    string msg = 9;
}